src/qiboconnection/connection.py

```diff
@@ -25,6 +25,7 @@
 import requests
 from typeguard import typechecked
 
+from qiboconnection import __version__ as VERSION
```

Comment:
Disable the `cyclic-import` pylint warning to be able to merge this PR!

```suggestion
from qiboconnection import __version__ as VERSION  # pylint: disable=cyclic-import
```